frontend/amundsen_application/static/js/pages/TableDetailPage/TableIssues/index.tsx

```diff
@@ -132,9 +139,9 @@ export class TableIssues extends React.Component<TableIssueProps> {
       return reportIssueLink;
     }
 
-    if (openIssuesUrl && closedIssuesUrl) {
-      return (
-        <span className="table-more-issues" key="more-issue-link">
+    return (
```

Comment:
The unrelated changes here were to fix a small bug that showed up if a table only had associated issues that were closed and hadn't been updated in over 30 days. The incorrect link was provided in that case and the user could not view the closed issues.